src/send/test.rs

```diff
@@ -122,6 +123,82 @@ fn without_optional_data() {
     assert_eq!(body, json!({ "messageId": "mock:deadbeef" }).to_string());
 }
 
+#[test]
+fn unicode_email_field() {
+    let client = setup();
+
+    let mut response = client
+        .post("/send")
+        .header(ContentType::JSON)
+        .body(
+            r#"{
+      "to": "시험@example.com",
+      "subject": "bar",
+      "body": {
+        "text": "baz"
+      },
+      "provider": "mock"
+    }"#,
+        )
+        .dispatch();
+
+    assert_eq!(response.status(), Status::Ok);
+
+    let body = response.body().unwrap().into_string().unwrap();
+    assert_eq!(body, json!({ "messageId": "mock:deadbeef" }).to_string());
+}
+
+#[test]
+fn unicode_message_body() {
+    let client = setup();
+
+    let mut response = client
+        .post("/send")
+        .header(ContentType::JSON)
+        .body(
+            r#"{
+      "to": "foo@example.com",
+      "subject": "bar",
+      "body": {
+        "text": "🦀 시험 🦀",
+        "html": "<p>🦀 시험 🦀</p>"
+      },
+      "provider": "mock"
+    }"#,
+        )
+        .dispatch();
+
+    assert_eq!(response.status(), Status::Ok);
+
+    let body = response.body().unwrap().into_string().unwrap();
+    assert_eq!(body, json!({ "messageId": "mock:deadbeef" }).to_string());
+}
+
+#[test]
+fn unicode_message_subject() {
+    let client = setup();
+
+    let mut response = client
+        .post("/send")
+        .header(ContentType::JSON)
+        .body(
+            r#"{
+      "to": "foo@example.com",
+      "subject": "🦀 시험 🦀",
+      "body": {
+        "text": "baz"
+      },
+      "provider": "mock"
+    }"#,
+        )
+        .dispatch();
+
+    assert_eq!(response.status(), Status::Ok);
+
+    let body = response.body().unwrap().into_string().unwrap();
+    assert_eq!(body, json!({ "messageId": "mock:deadbeef" }).to_string());
+}
```

Comment:
These tests are great but is it also possible to add some unicode tests to `src/providers/ses/test.rs`, to make sure that there's no surprises for us at that level?